test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -332,6 +332,12 @@ set(NNPA_TEST_LIST
     test_softmax_example_cpu,zdnn_softmax_ext,NO_DYNAMIC_SHAPE_TEST
     # test_softmax_large_number_cpu #  accuracy error
 
+    # ==OP== Softplus
+    # ==MIN== 1
+    # ==LIM== - `axis` must be the last dimension, i.e. `rank - 1` or -1.
```

Comment:
Sorry I happened to see this and wonder what `axis` means? I don't see `axis` in the definition of `softplus`:https://onnx.ai/onnx/operators/onnx__Softplus.html.

Reply:
Also, please update the document https://github.com/onnx/onnx-mlir/blob/main/docs/SupportedONNXOps-NNPA.md to include softplus. Thanks!

Reply:
@tungld I updated descriptions about Softplus limitation on NNPA. Thank you for the comments. I copied the limitation from Softmax op, and forgot to update it.